src/prisma/schema.prisma

```diff
@@ -595,6 +603,9 @@ model Vote {
   voteOptions   VoteOptions[]
   notifications VoteNotification[]
   evaluation    ProposalEvaluation?
+  chainId       Int?
+  tokenAddress  String?
+  blockNumber   Int?
```

Comment:
i wonder if we'd need to use a string. does snapshot use int for blocknumber?

Reply:
They use `string`! Great catch!. I will update it to string.